plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/BooleanWriteFunction.java

```diff
@@ -27,4 +27,28 @@ default Class<?> getJavaType()
 
     void set(PreparedStatement statement, int index, boolean value)
             throws SQLException;
+
+    static BooleanWriteFunction of(int jdbcType, BooleanWriteFunctionImplementation implementation)
```

Comment:
- jdbcType -> nullJdbcType
- `rnn` for `implementation` (this is a constructor)

Reply:
Done